.github/workflows/dependabot-automerge.yml

```diff
@@ -0,0 +1,20 @@
+name: Dependabot auto-merge
+on: pull_request_target
+permissions:
+  contents: read
+  pull-requests: write
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1.0.0
```

Comment:
What are we using the metadata for at this point?

Reply:
Yeah it's a bit obscure but it ends up checking the PR is from dependabot, has only one commit and is verified

Reply:
The action will error if those checks don't pass